### ISSUE TITLE
syntax error not reported, internal tables

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#1548]
following should give a syntax error, as left hand is a table right hand is simple

```abap
DATA integers TYPE STANDARD TABLE OF i WITH DEFAULT KEY.
integers = 2.
```
